src/main/java/foodwhere/logic/parser/ParserUtil.java

```diff
@@ -103,6 +107,9 @@ public static Address parseAddress(String address) throws ParseException {
     public static Tag parseTag(String tag) throws ParseException {
         requireNonNull(tag);
         String trimmedTag = tag.trim();
+        if (trimmedTag.isEmpty()) {
```

Comment:
I think we should also update the JavaDoc about this new behaviour. I would recommend making this private since it might be a confusing function to use.

Reply:
As per our discussion on 30/10/2022 this fix (PR) will be closed and the issue will be fixed with a docs update in UG.